optimum/intel/neural_compressor/neural_coder_adaptor.py

```diff
@@ -0,0 +1,26 @@
+class NeuralCoderAdaptor:
```

Comment:
It would be great to add a test as well to verify no modifications in the future is breaking anything

Reply:
Thanks for raising this. There's only two string constants in this Class and will not have more items in the future. Would you advice on how to add a test for only string constants? Thanks!

Reply:
Discussed with Haihao. We will add a test for this.

Reply:
Test added. @echarlaix

Reply:
Thanks @kaiyaointel !